internal/controlplane/handlers_githubwebhooks.go

```diff
@@ -296,42 +294,6 @@ func (s *Server) registerWebhookForRepository(
 	return regResult, nil
 }
 
-func (s *Server) deleteWebhookFromRepository(
-	ctx context.Context,
-	provider db.Provider,
-	dbrepo db.Repository,
-) error {
-	pbOpts := []providers.ProviderBuilderOption{
-		providers.WithProviderMetrics(s.provMt),
-		providers.WithRestClientCache(s.restClientCache),
-	}
-	providerBuilder, err := providers.GetProviderBuilder(ctx, provider, s.store, s.cryptoEngine, pbOpts...)
-	if err != nil {
-		return status.Errorf(codes.Internal, "cannot get provider builder: %v", err)
-	}
-
-	client, err := providerBuilder.GetGitHub()
```

Comment:
What happened to this providers-finding stuff?  Did moving it behind `repositories/github` mean that we already know the provider?

Reply:
The provider-getting logic lives in the unexported method I added into `handlers_repositories`